config.yaml

```diff
@@ -119,3 +119,9 @@ options:
         in this config will fall back to the value defined in `global-rps-limit`.
     default: ""
     type: string
+
+  db-tls-enabled:

```

Comment:
Question: is it not possible to get this information from the postgresql relation?

Reply:
You're absolutely right, I missed that the interface provides a tls flag in the relation databag. I've updated the logic to grab this flag from the relation rather than from the charm's config. Thanks!

Reply:
Coming back to this, it seems there's a bug with the postgresql-k8s charm that is causing it not to provide the `tls` flag in the relation databag. I've filed an issue [here](https://github.com/canonical/postgresql-k8s-operator/issues/698), and reverted to the config-based approach as a workaround for now.